### ISSUE TITLE
Installer should have flag to skip creation of RG/User/Namespace

### ISSUE THREAD

**Issue by padthaitofuhot** [EMCECS/ECS-CommunityEdition#77]

**Comment by padthaitofuhot** [comment]
How about we skip these steps when the stanzas don't exist in the deploy.yml ?

**Comment by adrianmo** [comment]
I vote for that too. It makes perfect sense and it's very intuitive.

**Comment by padthaitofuhot** [comment]
My quick patch for this doesn't quite work.  A few bits require the relevant configurations to be present in in deploy.yml.  The obvious two issues are:

1. I need to make the Ansible dynamic inventory script work right without these configs present.
```
ERROR! The file /usr/local/src/ui/inventory.py is marked as executable, but failed to execute correctly.
```
2. @adrianmo We need the validator to consider the items below optional.
```
Schema validation failed:
 - Cannot find required key 'virtual_data_centers'. Path: '/facts'.
 - Cannot find required key 'replication_group_defaults'. Path: '/facts'.
 - Cannot find required key 'storage_pools'. Path: '/facts'.
 - Cannot find required key 'virtual_data_center_defaults'. Path: '/facts'.
 - Cannot find required key 'storage_pool_defaults'. Path: '/facts'.
 - Cannot find required key 'replication_groups'. Path: '/facts'.
```

**Comment by padthaitofuhot** [comment]
@adrianmo Also, I'm dumb - SPs and VDCs are required.  RGs and up are not.  Sorry for the spam.

**Comment by padthaitofuhot** [comment]
Resolved by #231
